src/Goteo/Library/Forms/Model/ImpactDataForm.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * This file is part of the Goteo Package.
+ *
+ * (c) Platoniq y Fundación Goteo <fundacion@goteo.org>
+ *
+ * For the full copyright and license information, please view the README.md
+ * and LICENSE files that was distributed with this source code.
+ */
+
+namespace Goteo\Library\Forms\Model;
+
+use Goteo\Library\Forms\AbstractFormProcessor;
+use Goteo\Library\Forms\FormModelException;
+use Goteo\Library\Text;
+use Goteo\Model\Image;
+use Goteo\Model\ImpactData;
+use Goteo\Util\Form\Type\DropfilesType;
+use Goteo\Util\Form\Type\TextType;
+use Goteo\Util\Form\Type\TextareaType;
+use Goteo\Util\Form\Type\SubmitType;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Validator\Constraints;
+
+class ImpactDataForm extends AbstractFormProcessor {
+
+    public function createForm() {
+
+        $model = $this->getModel();
+        $builder = $this->getBuilder();
+        $options = $builder->getOptions();
+        $data = $options['data'];
+
+        $builder
+            ->add('title', TextType::class, array(
+                'label' => 'regular-title',
+                'required' => true,
```

Comment:
By default, Symfony Form fields are required, so it's not needed to write this down: https://symfony.com/doc/current/forms.html#the-required-option